libnavigation-ui/src/main/java/com/mapbox/navigation/ui/feedback/FeedbackBottomSheet.java

```diff
@@ -42,26 +51,43 @@
 public class FeedbackBottomSheet extends BottomSheetDialogFragment implements Animator.AnimatorListener {
 
   public static final String TAG = FeedbackBottomSheet.class.getSimpleName();
+  public static final int FEEDBACK_MAIN_FLOW = 0;
```

Comment:
may be make `@IntDef` for `FEEDBACK_MAIN_FLOW` and `FEEDBACK_DETAIL_FLOW`

Reply:
Fixed